x/committee/legacy/v0_15/types.go

```diff
@@ -0,0 +1,32 @@
+package v0_15
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// GenesisState is state that must be provided at chain genesis.
+type GenesisState struct {
+	NextProposalID uint64     `json:"next_proposal_id" yaml:"next_proposal_id"`
+	Committees     Committees `json:"committees" yaml:"committees"`
+	Proposals      []Proposal `json:"proposals" yaml:"proposals"`
+	Votes          []Vote     `json:"votes" yaml:"votes"`
+}
+
+func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
```

Comment:
nit: CommitteeChange or CommitteeDelete proposals aren't registered on the codec used to unmarshal the v0.15 genesis in the top level migration.

Reply:
good point, i added those back in since its pretty straightforward https://github.com/Kava-Labs/kava/pull/1079/commits/1e7a46c23d41300c3e01a0baa0c0f6f7118ec5c5